### PR TITLE
Dev 3994 make the ai branch audits beautiful

### PR DESCRIPTION
**Pull request by eslopfer**

### CODE REVIEW

File: subjects/ai/train-and-evalute-machine-learning-models/audit/README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+##### The exercise is validated if all questions of the exercise are validated.
 
 ##### Activate the virtual environment. If you used `conda` run `conda activate your_env`.
 
 ##### Run `python --version`.
 
 ###### Does it print `Python 3.x`? x >= 8
 
-##### Does `import jupyter`, `import numpy`, `import pandas`, `import matplotlib` and `import sklearn` run without any error?
+##### Do `import jupyter`, `import numpy`, `import pandas`, `import matplotlib` and `import sklearn` run without any error?
```
Comment (mikysett) [inline subjects/ai/train-and-evalute-machine-learning-models/audit/README.md:11]: should be an `h6`

File: subjects/ai/visualizations/audit/README.md [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
+##### The exercise is validated if all questions of the exercise are validated
 
 ##### Activate the virtual environment. If you used `conda` run `conda activate your_env`
 
 ##### Run `python --version`
 
 ###### Does it print `Python 3.x`? x >= 8
 
-##### Does `import jupyter`, `import numpy`, `import pandas`, `matplotlib` and `plotly` run without any error ?
+##### Do `import jupyter`, `import numpy`, `import pandas`, `matplotlib` and `plotly` run without any error ?
```
Comment (mikysett) [inline subjects/ai/visualizations/audit/README.md:11]: should be an `h6`, also there is an extra space before `?`

**Review by mikysett — CHANGES_REQUESTED** [review]
Overall everything seems well formatted, there is few questions in the wrong format commented inline.
There is some grammar errors but I guess they will be reviewed in the language review, I've pointed out only the ones that can prevent the understanding of the question.

Also as a recurrent typo easy to fix there is this sentence repeated 10 times:
"The exercise is validated is all questions of the exercise are validated"
It could be nice if you can change the second `is` into an `if` :-)

File: subjects/ai/credit-scoring/audit/README.md [hunk 3]
```diff
@@ -34,54 +34,54 @@ project
...
+    - Visualisations that show the comparison between this client and other clients.
 
 ##### SHAP values on the model are displayed through a summary plot that shows the important features and their impact on the target. This is optional if you have already computed the features importance.
 
-###### Do the 3 clients are selected as expected ? 2 clients from the train set (1 on which the model is correct and 1 on which the model's wrong) and 1 client from the test set.
+###### Are the 3 clients selected as expected? 2 clients from the train set (1 on which the model is correct and 1 on which the model's wrong) and 1 client from the test set.
 
-##### SHAP values on predictions are computed for the 3 clients. The force plot shows what variables contributes the most to the score. **Check that the score outputted by the force plot corresponds to the one outputted by the model.**
+###### SHAP values on predictions are computed for the 3 clients. The force plot shows what variables contributes the most to the score. **Check that the score outputted by the force plot corresponds to the one outputted by the model.**
```
[the remark this answers is not in the document]
Comment (eslopfer) [reply subjects/ai/credit-scoring/audit/README.md:87]: Thanks. Done.

File: subjects/ai/natural-language-processing-with-spacy/audit/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #### Exercise 0: Environment and libraries
 
-##### The exercise is validated is all questions of the exercise are validated.
+##### The exercise is all questions of the exercise are validated.
```
[the remark this answers is not in the document]
Comment (eslopfer) [reply subjects/ai/natural-language-processing-with-spacy/audit/README.md:3]: Thanks, fixed.

File: subjects/ai/model-selection-methodology/audit/README.md [hunk 5]
```diff
@@ -81,7 +84,7 @@ gridsearch.fit(X_train, y_train)
 
 The answers that uses another list of parameters are accepted too !
 
-##### The question 2 is validated if these attributes were used:
+###### For question 2, whe these attributes used?
```
[the remark this answers is not in the document]
Comment (eslopfer) [reply subjects/ai/model-selection-methodology/audit/README.md:87]: Thanks, corrected.

File: subjects/ai/train-and-evalute-machine-learning-models/audit/README.md (hunk 1 above)
Comment (eslopfer) [reply subjects/ai/train-and-evalute-machine-learning-models/audit/README.md:11]: Thanks. Done

File: subjects/ai/visualizations/audit/README.md (hunk 2 above)
Comment (eslopfer) [reply subjects/ai/visualizations/audit/README.md:11]: Thanks, corrected :grin:

**Comment by eslopfer** [comment]
Thanks for your review, this PR was huge. I still have to fix the conflicts but first I would like to discuss something with Dav. :pray:

File: subjects/ai/natural-language-processing-with-spacy/audit/README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 #### Exercise 0: Environment and libraries
 
-##### The exercise is all questions of the exercise are validated.
+##### The exercise is is validated if all questions of the exercise are validated.
```
Comment (mikysett) [inline subjects/ai/natural-language-processing-with-spacy/audit/README.md:3]: `is` repeated twice

**Review by mikysett — APPROVED** [review]
There is only one small typo that I've spotted in the last commit.
I approve the PR so you can feel free to mark the only conversation still opened as resolved when it will be fixed, this should help moving faster.

File: subjects/ai/natural-language-processing-with-spacy/audit/README.md (hunk 6 above)
Comment (eslopfer) [reply subjects/ai/natural-language-processing-with-spacy/audit/README.md]: Thanks, done.

**Pull request merged**